internal/dag/httpproxy_processor.go

```diff
@@ -297,6 +297,18 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 				} else {
 					svhost.AuthorizationResponseTimeout = timeout
 				}
+
+				if auth.BufferSettings != nil {
+					var maxRequestBytes uint32 = 1024
```

Comment:
note that this logic adds a default in the processing code, in addition to a default we may add in the kubebuilder defaults, just have to keep in mind in case we change the default

Reply:
Yes, if we're adding a default, it should be in the kubebuilder settings, and if we do have a default in processing, it should be in a constant or something somewhere, so there's an obvious place to change it.

Reply:
Added a const to this file. 
I didn't see a common place to add constants.